libs/ui-lib/lib/common/api/assisted-service/NewFeatureSupportLevelsAPI.ts

```diff
@@ -9,10 +9,11 @@ const NewFeatureSupportLevelsAPI = {
     return `/v2/support-levels`;
   },
 
-  featuresSupportLevel(openshiftVersion: string, cpuArchitecture?: string) {
+  featuresSupportLevel(openshiftVersion: string, cpuArchitecture?: string, platformType?: string) {
     let queryParams = '?';
     queryParams += openshiftVersion ? `openshift_version=${openshiftVersion}&` : '';
-    queryParams += cpuArchitecture ? `cpu_architecture=${cpuArchitecture}` : '';
+    queryParams += cpuArchitecture ? `cpu_architecture=${cpuArchitecture}&` : '';
+    queryParams += platformType ? `platform_type=${platformType}` : '';
     return clientWithoutCaseConverter.get<FeaturesSupportsLevel>(
       `${NewFeatureSupportLevelsAPI.makeBaseURI()}/features${queryParams}`,
     );
```

Comment:
Can you please try this instead of serializing the query-string manually?
```ts
featuresSupportLevel(openshiftVersion: string, cpuArchitecture?: string, platformType?: string) {
    return clientWithoutCaseConverter.get<FeaturesSupportsLevel>(
      `${NewFeatureSupportLevelsAPI.makeBaseURI()}/features`,
      {
        params: {
          openshift_version: openshiftVersion,
          cpu_architecture: cpuArchitecture,
          platform_type: platformType,
        }
      }
    );
  },
```

Reply:
it works